### PR TITLE
Wip v1 add subject register

### PR DESCRIPTION
**Pull request by SebasVel16**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,6 +30,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
```
Comment (cacastanedal) [inline pom.xml:35]: what does this module do?

File: pom.xml [hunk 2]
```diff
@@ -40,6 +44,10 @@
 			<artifactId>postgresql</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-validation</artifactId>
```
Comment (cacastanedal) [inline pom.xml:49]: what does this module do?

**Review by cacastanedal — APPROVED** [review]
Great, just a couple of questions left.

Nice work

File: pom.xml (hunk 2 above)
Comment (SebasVel16) [reply pom.xml:49]: This one imports the annotations to validate the DTO objects information, with the @Valid tag, tags like @NotNull, @Max, @Min, etc

File: pom.xml (hunk 1 above)
Comment (SebasVel16) [reply pom.xml:35]: I'm not really sure that one, I believed it was to start any of the dependencies I use but I realized that there are other dependencies that already start services like JPA or Web. Maybe this one is not necesary, I'll fix it

**Pull request merged**